app/gitcoinbot/actions.py

```diff
@@ -118,11 +210,21 @@ def create_token(install_id):
     return token
 
 
-def determine_response(owner, repo, comment_id, comment_text, issue_id, install_id):
+def get_text_from_query_responses(comment_text, sender):
+    """Based on comment_text an assigned response is returned"""
+    comment_text_low = comment_text.casefold()
+    result_text = GitcoinBotResponses.objects.filter(request=comment_text_low)
+    if result_text.exists():
+        return f'@{sender} {result_text.first().response}'
+    return ""
```

Comment:
Can you make this consistent quoting? `return ''` or `return f"@{sender} {result_text.first().response}"`